compliance/report.py

```diff
@@ -120,9 +120,9 @@ def generate_toc(self, rpt_metadata):
                 continue
             rpt_descr = meta['description'] or pathlib.Path(rpt).name
             rpt_url = self.locker.get_remote_location(rpt, False)
-            check = meta['checks'][0].rsplit('.', 1).pop(0)
+            check = meta.get('checks', ['N/A'])[0].rsplit('.', 1).pop(0)
```

Comment:
Would a None or empty list work?

Reply:
This way "N/A" finds its way to the template and it is displayed as "Check: N/A" in the TOC rather than nothing.